demosplan/DemosPlanCoreBundle/Command/Data/DeleteProcedureCommand.php

```diff
@@ -540,8 +540,12 @@ private function repopulateElasticsearch(): void
             return;
         }
 
+        $env = $this->parameterBag->get('kernel.environment');
+        $this->output->writeln("Repopulating ES with env: $env");
+
+        $repopulateEsCommand = 'dev' === $env ? 'dplan:elasticsearch:populate' : 'dplan:elasticsearch:populate -e prod';
```

Comment:
Nitpick: you could even add `--no-debug` to add some more performance

Reply:
Done in [1e10b10](https://github.com/demos-europe/demosplan-core/pull/1958/commits/1e10b1063a5e74cc76a420fd31aa1e1c8a12234b)